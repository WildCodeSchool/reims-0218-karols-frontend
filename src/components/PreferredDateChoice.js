import React from "react"

const PreferredDateChoice = () => {
  return (
    <div className="text-center mb-5">
      <form>
        <p>Sélectionnez votre date souhaitée :</p>
        <input type="date" name="preferreddatechoice" />
      </form>
    </div>
  )
}

export default PreferredDateChoice
