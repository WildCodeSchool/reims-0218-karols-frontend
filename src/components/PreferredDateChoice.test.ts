import React from "react"
import ReactDOM from "react-dom"
import PreferredDateChoice from "./PreferredDateChoice"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<PreferredDateChoice />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<PreferredDateChoice />).toJSON()
  expect(tree).toMatchSnapshot()
})
