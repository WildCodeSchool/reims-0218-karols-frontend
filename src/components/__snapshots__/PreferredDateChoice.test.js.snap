// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text-center mb-5"
>
  <form>
    <p>
      Sélectionnez votre date souhaitée :
    </p>
    <input
      name="preferreddatechoice"
      type="date"
    />
  </form>
</div>
`;
